src/src/PluginStructs/P093_data_struct.cpp

```diff
@@ -362,6 +378,29 @@ void P093_data_struct::applySettings() {
   packet[21] = checkSum(packet, 21);
 
   sendPacket(packet, PACKET_LEN);
+
+  if (_writeStatus.isDirty(RemoteTemperature)) {
+    packet[PACKET_LEN] = {};
+
+    packet[5] |= 0x07;
+    if(_wantedSettings.remoteTemperature > 0) {
+      packet[6] |= 0x01;
+      if (_tempMode) {
+        packet[8] = static_cast<uint8_t>(_wantedSettings.remoteTemperature * 2.0f + 128.0f);
+      } else {
+        packet[7] = _wantedSettings.remoteTemperature;
+      }
+    }
+    else {
+      packet[6] |= 0x00;
```

Comment:
Is this intended?
A logic OR with 0 doesn't change anything.

Reply:
No it´s not intended, will be fixed...